generators/liquibase/templates/src/main/resources/config/liquibase/changelog/updated_entity.xml.ejs

```diff
@@ -25,15 +25,15 @@
     <changeSet id="<%= changelogDate %>-1-add-columns" author="jhipster">
         <addColumn tableName="<%= entity.entityTableName %>">
     <%_ for (field of addedFields) { _%>
-            <column name="<%= field.columnName %>" type="<%= field.columnType %>"<%- this.formatAsLiquibaseRemarks(field.documentation, true) %>/>
+            <column name="<%= field.columnName %>" type="<%= field.columnType %>"<%- include('./default-field-value', {field, inline: true}) -%><%- this.formatAsLiquibaseRemarks(field.documentation, true) %>/>
```

Comment:
I prefer a property prepared in the generator instead of a partial template for this use case.
```suggestion
            <column name="<%= field.columnName %>" type="<%= field.columnType %>"<%- field.liquibaseDefaultValueTagAndValue %><%- this.formatAsLiquibaseRemarks(field.documentation, true) %>/>
```

Reply:
Ok, I'll adjust it accordingly :-)

Reply:
The problem about methods is that the template becomes dependent on the generator api.
 - the template cannot be reused by side-by-side blueprints unless the blueprint extends the original generator.
 - side-by-side blueprints cannot customize methods calls, it can customize properties.

So we should avoid method calls inside templates and prefer prepared/computed properties.